### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by FinalAngel**
besides translations also aligned some code cleanup

### CODE REVIEW

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
 msgstr ""
 
 #: models.py:28
-msgid "right"
+msgid "Align right"
 msgstr ""
 
 #: models.py:29
-msgid "center"
+msgid "Align center"
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *centre* if we want to use Bringlish. But *center* to match CSS.

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
-#: models.py:32
-msgid "image"
+#: models.py:37
+msgid "Open in new window"
 msgstr ""
 
-#: models.py:34
-msgid "link"
+#: models.py:38
+msgid "Open in same window"
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *this window* or *the same window*, whichever makes more sense in the context

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
 
 #: models.py:29
-msgid "center"
+msgid "Align center"
 msgstr ""
 
-#: models.py:32
-msgid "image"
+#: models.py:37
+msgid "Open in new window"
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *a new window*

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
-#: models.py:35
-msgid "If present, clicking on image will take user to link."
+#: models.py:39
+msgid "Delegate to parent"
 msgstr ""
 
-#: models.py:38
-msgid "page"
+#: models.py:40
+msgid "Delegate to top"
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: I don't know what "Delegate to parent" (or top) might mean, so I am not sure if this is correct or not.

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
+msgid "External image"
 msgstr ""
 
-#: models.py:45
+#: models.py:78
 msgid ""
-"Specifies an alternate text for an image, if the imagecannot be displayed."
-"<br />Is also used by search enginesto classify the image."
+"If provided, overrides the embedded image. Certain options such as cropping "
+"are not applicable for external images."
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *applicable to*

Also better to say exactly which options, not just "certain options".

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
-msgid "long description"
+#: models.py:82
+msgid "Width"
 msgstr ""
 
-#: models.py:51
-msgid "When user hovers above picture, this text will appear in a popup."
+#: models.py:85
+msgid ""
+"The image width as number in pixels. Example: \"720\" and not \"720px\"."
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *The image width in pixels.*

I don't think it's necessary to say more.

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
-msgid "side"
+#: models.py:89
+msgid "Height"
 msgstr ""
 
-#: models.py:56
-msgid "Move image left, right or center."
+#: models.py:92
+msgid ""
+"The image height as number in pixels. Example: \"720\" and not \"720px\"."
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *The image height in pixels.*

I don't think it's necessary to say more.

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
-#: models.py:58
-msgid "width"
+#: models.py:96
+msgid "Alignment"
 msgstr ""
 
-#: models.py:59 models.py:61
-msgid "Pixel"
+#: models.py:100
+msgid "Aligns the image to the selected option."
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *... according to the selected option*?

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
+msgid "Outputs the raw image without cropping."
+msgstr ""
+
+#: models.py:156
+msgid "Crop image"
+msgstr ""
+
+#: models.py:159
+msgid ""
+"Crops the image according to the given thumbnail settings in the template."
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *according to the thumbnail settings provided in the template*

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
+msgstr ""
+
+#: models.py:162
+msgid "Upscale image"
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: I would prefer *Scale up image*.

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
+msgstr ""
+
+#: models.py:165
+msgid ""
+"Upscales the image to the size of the thumbnail settings in the template."
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: I would prefer *Scales up image*.

File: djangocms_picture/locale/en/LC_MESSAGES/django.po (hunk 2 above)
Comment (FinalAngel) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: this is added to 3 different addons like that, will leave it for now :(

File: djangocms_picture/locale/en/LC_MESSAGES/django.po (hunk 4 above)
Comment (FinalAngel) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: same here...

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 12]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
+msgstr ""
+
+#: models.py:171
+msgid "Thumbnail options"
+msgstr ""
+
+#: models.py:174
+msgid "Overrides width, height, crop and upscale with the provided preset."
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: I am not sure what this should be, but either:

> Overrides width, height, and crop; scales up to the provided preset dimensions.

or 

> Override width, height, crop; scale up to the provided preset dimensions.

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 13]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
+msgstr ""
+
+#: models.py:197
+msgid "<file is missing>"
+msgstr ""
+
+#: models.py:245
+msgid ""
+"You have defined an external and internal link. Only one option is allowed."
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *You have defined both external and internal links.*

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 14]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
+msgstr ""
+
+#: models.py:252
+msgid "You need to add either an image or an external link to an image."
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *You need to add either an image, or a URL linking to an external image.*

File: djangocms_picture/locale/en/LC_MESSAGES/django.po [hunk 15]
```diff
@@ -17,80 +17,188 @@ msgstr ""
...
+msgstr ""
+
+#: models.py:276
+#, python-brace-format
+msgid ""
+"The cropping selection is not valid. You cannot combine \"{field_a}\" with "
```
Comment (evildmp) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: *"Invalid cropping settings. You cannot...*

File: djangocms_picture/migrations/0004_adapt_fields.py [hunk 16]
```diff
@@ -33,7 +33,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='picture',
             name='link_target',
-            field=models.CharField(blank=True, max_length=255, verbose_name='Link target', choices=[('_blank', 'Open in new window.'), ('_self', 'Open in same window.'), ('_parent', 'Delegate to parent.'), ('_top', 'Delegate to top.')]),
+            field=models.CharField(blank=True, max_length=255, verbose_name='Link target', choices=[('_blank', 'Open in new window'), ('_self', 'Open in same window'), ('_parent', 'Delegate to parent'), ('_top', 'Delegate to top')]),
```
Comment (evildmp) [inline djangocms_picture/migrations/0004_adapt_fields.py]: See note above about new/same window wording; needs to match.

File: djangocms_picture/models.py [hunk 17]
```diff
@@ -34,10 +34,10 @@
...
 
 LINK_TARGET = (
-    ('_blank', _('Open in new window.')),
-    ('_self', _('Open in same window.')),
-    ('_parent', _('Delegate to parent.')),
-    ('_top', _('Delegate to top.')),
+    ('_blank', _('Open in new window')),
+    ('_self', _('Open in same window')),
+    ('_parent', _('Delegate to parent')),
+    ('_top', _('Delegate to top')),
```
Comment (evildmp) [inline djangocms_picture/models.py]: See note above re. new/same and delegation.

File: djangocms_picture/locale/en/LC_MESSAGES/django.po (hunk 1 above)
Comment (FinalAngel) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: it does add css "centre"

File: djangocms_picture/locale/en/LC_MESSAGES/django.po (hunk 3 above)
Comment (FinalAngel) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: Same here, standard for all addons currently

File: djangocms_picture/locale/en/LC_MESSAGES/django.po (hunk 6 above)
Comment (FinalAngel) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: Examples are always good, having tickets where people complained

File: djangocms_picture/locale/en/LC_MESSAGES/django.po (hunk 7 above)
Comment (FinalAngel) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: same here

File: djangocms_picture/locale/en/LC_MESSAGES/django.po (hunk 10 above)
Comment (FinalAngel) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: Its also referred in Filer like that

File: djangocms_picture/locale/en/LC_MESSAGES/django.po (hunk 11 above)
Comment (FinalAngel) [inline djangocms_picture/locale/en/LC_MESSAGES/django.po]: Its also referred in Filer like that

File: djangocms_picture/models.py (hunk 17 above)
Comment (FinalAngel) [inline djangocms_picture/models.py]: same here, now a standard :D

**Pull request merged**